RealFlight/Released_Models/Planes/DG-800_Glider/README.md

```diff
@@ -0,0 +1,41 @@
+DG-800 Glider
```

Comment:
this needs to be in markdown format to display properly

Reply:
fixed....finally figured out how to preview :use my repo on github and edit there